migrator.go

```diff
@@ -0,0 +1,84 @@
+package influxdb
+
+import "context"
+
+// MigrationManager is the manager of migrators.
+type MigrationManager struct {
```

Comment:
I dont think we will want more then one MigrationManger per execution of the binary. I would make this work similar to how http deals with adding new handlers (https://golang.org/src/net/http/server.go?s=74743:74787#L2424). I would even go as far to say that it would be best to make the migration manager private and expose a `Register(migrator Migrator)` and a `Migrate` function